app/app.go

```diff
@@ -150,9 +150,9 @@ func (a *App) TelemetryId() string {
 	return a.Srv().TelemetryId()
 }
 
-func (s *Server) HTMLTemplates() *template.Template {
+func (s *Server) TemplatesContainer() *templates.Container {
 	if s.htmlTemplateWatcher != nil {
-		return s.htmlTemplateWatcher.Templates()
+		return s.htmlTemplateWatcher
 	}
 
 	return nil
```

Comment:
This can return nil if the watcher is not set, or closed. In that case, calls using this will panic. We need to handle the nil case.